neo/rawio/neuronexusrawio.py

```diff
@@ -219,21 +219,10 @@ def _parse_header(self):
         # Add the minimum annotations
         self._generate_minimal_annotations()
 
-        # date comes out as:
-        # year-month-daydayofweektime all as a string so we need to prep it for
-        # entering into datetime
-        # example: '2024-07-01T13:04:49.4972245-04:00'
-        stringified_date_list = self.metadata['status']['start_time'].split('-')
-        year = int(stringified_date_list[0])
-        month = int(stringified_date_list[1])
-        day = int(stringified_date_list[2][:2]) # day should be first two digits of the third item in list
-        time_info = stringified_date_list[2].split(':')
-        hour = int(time_info[0][-2:])
-        minute = int(time_info[1])
-        second = int(float(time_info[2]))
-        microsecond = int(1000 * 1000 * (float(time_info[2]) - second))# second -> micro is 1000 * 1000
-
-        rec_datetime = datetime.datetime(year, month, day, hour, minute, second, microsecond)
+        # date comes out as: '2024-07-01T13:04:49.4972245-04:00' so in ISO format
+        datetime_string = self.metadata["status"]["start_time"]
+        rec_datetime = datetime.datetime.fromisoformat(datetime_string)
+
```

Comment:
what if we do
try-except where we cut off the trailing -4:00 which I think was support < 3.11.